src/custom-types.json

```diff
@@ -637,6 +637,14 @@
   "image/x-xcf": {
     "compressible": false
   },
+  "image/dds" : {
+    "sources": [
+      "https://docs.microsoft.com/en-us/windows/win32/wic/dds-format-overview",
+      "https://en.wikipedia.org/wiki/DirectDraw_Surface"
```

Comment:
You can remove this, as the sources need to be primary courses only, no Wikipedia links (https://github.com/jshttp/mime-db#contributing).

Reply:
Tahnk you I'll fix this immediately